maps/interiors/arc.dmm

```diff
@@ -1,296 +1,112 @@
-	},
-/obj/structure/machinery/computer/overwatch/almayer/small{
+/obj/structure/machinery/computer/groundside_operations/arc{
 	dir = 4;
-	layer = 3.01;
-	pixel_y = -5
-	},
-/turf/open/shuttle/vehicle{
-	icon_state = "floor_3_10_1"
-	},
-/area/interior/vehicle/arc)
-"Gy" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "front_1"
-	},
-/turf/open/void/vehicle,
-/area/space)
-"HB" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "corner_small_L"
-	},
-/obj/structure/interior_wall/apc{
-	icon_state = "rear_5"
-	},
-/turf/open/void/vehicle,
-/area/space)
-"HJ" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "front_6"
-	},
-/turf/open/void/vehicle,
-/area/space)
-"Jc" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "corner_small_R"
+	layer = 2.981;
+	pixel_x = -13;
+	pixel_y = -9
 	},
-/obj/structure/interior_wall/apc{
-	icon_state = "front_5"
-	},
-/turf/open/void/vehicle,
-/area/space)
-"Ng" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "wall"
-	},
-/turf/open/void/vehicle,
-/area/space)
-"NS" = (
-/obj/structure/bed/chair/comfy{
+/obj/structure/bed/chair/comfy/arc{
 	dir = 8
 	},
-/turf/open/shuttle/vehicle{
-	icon_state = "floor_3_5"
-	},
+/turf/open/floor,
 /area/interior/vehicle/arc)
-"Ol" = (
-/obj/effect/landmark/interior/spawn/entrance{
-	alpha = 50;
-	exit_type = /obj/structure/interior_exit/vehicle/apc;
-	name = "Right APC door";
-	tag = "right"
-	},
-/obj/effect/landmark/interior/spawn/interior_viewport{
-	dir = 8;
-	pixel_x = 5;
-	pixel_y = 1
-	},
-/turf/open/shuttle/vehicle{
-	icon_state = "floor_1_12"
-	},
-/area/interior/vehicle/arc)
-"Po" = (
-/obj/structure/surface/table/reinforced/prison{
-	pixel_y = -3
-	},
-/obj/structure/machinery/computer/intel/disk_reader{
+"DG" = (
+/obj/structure/machinery/computer/overwatch/almayer/small{
 	dir = 4;
-	pixel_y = 6
+	layer = 2.982;
+	pixel_x = -13;
+	pixel_y = 10
 	},
-/obj/structure/machinery/computer/groundside_operations{
+/obj/structure/machinery/computer/intel/disk_reader{
 	dir = 4;
-	pixel_y = -7
-	},
-/turf/open/shuttle/vehicle{
-	icon_state = "floor_3_8_1"
-	},
-/area/interior/vehicle/arc)
-"Rb" = (
-/turf/open/void/vehicle,
-/area/space)
-"Rf" = (
-/obj/structure/interior_wall/apc{
-	icon_state = "front_2"
+	layer = 2.983;
+	pixel_x = -13;
+	pixel_y = -3
 	},
-/turf/open/void/vehicle,
-/area/space)
-"Zs" = (
-/obj/structure/bed/chair/comfy{
+/obj/structure/bed/chair/comfy/arc{
 	dir = 8
 	},
-/turf/open/shuttle/vehicle{
-	icon_state = "floor_3_8_1"
+/turf/open/floor,
+/area/interior/vehicle/arc)
+"Ly" = (
+/obj/effect/landmark/interior/spawn/weapons_loader{
+	pixel_x = -2
 	},
+/turf/open/floor,
 /area/interior/vehicle/arc)
-"ZF" = (
+"NS" = (
 /obj/effect/landmark/interior/spawn/interior_viewport{
-	dir = 8;
-	pixel_x = 5;
-	pixel_y = 8
-	},
-/obj/effect/landmark/interior/spawn/entrance{
-	alpha = 50;
-	dir = 1;
-	exit_type = /obj/structure/interior_exit/vehicle/apc;
-	name = "Left APC door";
-	pixel_y = 32;
-	tag = "left"
+	layer = 2.98;
```

Comment:
Layering looks to be incorrect: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/7036e83d-5a2b-4010-b629-09dbf076ed25)

Reply:
Also slightly funny to me 3 of the 4 computers to the left all can open tacmap but its fine.

Reply:
fixed

Reply:
Still appears to be an issue:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/e4e5fb77-3fe1-40b8-b1d5-6fa2bb907b9b)

Reply:
Fixed for real this time